src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs

```diff
@@ -77,24 +77,26 @@
                         </label>
 
                         <span class="help-inline-checkbox">
-                            <i class="icon-sonarr-form-info" title="Do you want to get notifications when episodes are upgraded to a better quality?"/>
+                            <i class="icon-sonarr-form-info" title="Be notified when episodes are upgraded to a better quality"/>
```

Comment:
I'm still struggling with the "Be notified" portion of this, thoughts on replacing it with "Notifications", Should it would becomes "Notifications when X".